### DISCUSSION TITLE
Installation without using NPM to Redhat server

### DISCUSSION THREAD

**Discussion by SafiuddinIsmail** [ConsenSys-archive/truffle#5875] [Installation]
Hi everyone, I would like to install Truffle and Ganache into my Redhat server, the problem is that we cannot install via NPM or any executable files because of company's policy, it had to be done by tar.gz and export the path of the folder to $PATH. I've managed to do it with geth but it doesn't work with Truffle suite. I've found https://github.com/trufflesuite/truffle/releases but it doesn't show any .tar.gz files that I can use and extract to my server. Can anyone point me to the right direction?

**Comment by FlaviuWeb3** [comment]
Hi @SafiuddinIsmail you can find the .tar.gz link for our truffle release here -> https://github.com/trufflesuite/truffle/archive/refs/tags/v5.7.9.tar.gz
